src/metabase/lib/drill_thru.cljc

```diff
@@ -109,4 +111,6 @@
     stage-number :- :int
     drill        :- ::lib.schema.drill-thru/drill-thru
     & args]
+   (log/infof "Applying drill thru: %s"
```

Comment:
This looks like debug level?

Reply:
Agree, but debug level doesn't pop up in the browser console and the main reason I want this is so I can debug it in the browser/Cypress tests. Is there some way to configure the log level from the browser console?